src/implementation/Client/GRPCClient/GRPCClient.ts

```diff
@@ -28,15 +28,18 @@ export default class GRPCClient implements IClient {
   private readonly client: GrpcDaprClient;
   private readonly clientCredentials: grpc.ChannelCredentials;
   private readonly logger: Logger;
+  public readonly grpcClientOptions: Partial<grpc.ClientOptions>;
```

Comment:
This should be private

Reply:
@XavierGeerinck This is the property where we store grpc metadata as interceptors (specifically, the dapr api token) and we need to be able to access it in the Proxy builder (`src/implementation/Client/GRPCClient/GRPCClientProxy.ts`) so we don't override it.